nodejs/week1/meal_sharing_website/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,17 @@
+const jsonMealsData = require("../data/meals.json");
+const jsonReviewsData = require("../data/reviews.json");
+
+const cheapMealsTest = function (request, response) {
+    
+    // This logic is to include reviews json data inside meals json data
+    const mealsInReview = jsonMealsData.map(item => {
+        item.reviews = jsonReviewsData.filter(element => element.mealId === item.id)
+        return item;
+    });
+
+    const cheapMealsReturn = mealsInReview.filter(item => item.price < 15);
+
+    response.json(cheapMealsReturn);
```

Comment:
not sure what this does, but last time i checked .json() doesn't take any arguments?[https://developer.mozilla.org/en-US/docs/Web/API/Body/json]( https://developer.mozilla.org/en-US/docs/Web/API/Body/json)

What are you trying to do here?

Reply:
I was trying to do this 
https://flaviocopes.com/express-send-json-response/